tests/catch/unit/memory/hipArrayCreate.cc

```diff
@@ -32,6 +32,15 @@ static constexpr auto NUM_H{4};
 static constexpr auto BIGNUM_H{100};
 static constexpr auto ARRAY_LOOP{100};
 
+#ifdef __HIP_PLATFORM_NVIDIA__
+#define HIP_TEX_REFERENCE hipTexRef
+#define HIP_ARRAY hiparray
+#else  // __HIP_PLATFORM_NVIDIA__
+#define ARRAY_DESTROY(array) HIPCHECK(hipFreeArray(array));
```

Comment:
ARRAY_DESTROY needs to be defined for nvidia platform as well. Otherwise it causes the test to fail in building on nvidia platform.